### PR TITLE
jira -> issue types

### PR DESCRIPTION
**Pull request by lwsanty**
close [[DEV-1307]]

Signed-off-by: lwsanty <lwsanty@gmail.com>

[DEV-1307]: https://athenianco.atlassian.net/browse/DEV-1307

### CODE REVIEW

File: jira/database/database_issue_type_pg.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+	"fmt"
+
+	"github.com/athenianco/cloud-common/dbs"
+)
+
+const issueTypeColumns = `acc_id, project_id, id, name, description, icon_url, is_subtask, avatar_id`
+
+func (db *pgDatabase) CreateIssueType(ctx context.Context, i *IssueType) error {
+	_, err := db.ExecContext(ctx, `INSERT INTO
+		jira.issue_type(acc_id, project_id, id, name, description, icon_url, is_subtask, avatar_id)
```
Comment (dennwc) [inline jira/database/database_issue_type_pg.go:15]: Use `issueTypeColumns`?

File: sql/migrations/080_jira_issue_type.up.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+CREATE TABLE jira.issue_type (
+    acc_id bigint NOT NULL,
+    project_id text NOT NULL,
+    id text NOT NULL,
```
Comment (dennwc) [inline sql/migrations/080_jira_issue_type.up.sql:4]:
```suggestion
    id text NOT NULL,
    project_id text NOT NULL,
```

**Review by dennwc — CHANGES_REQUESTED** [review]

File: jira/database/database_issue_type_pg.go (hunk 1 above)
Comment (lwsanty) [reply jira/database/database_issue_type_pg.go]: Done.

File: sql/migrations/080_jira_issue_type.up.sql (hunk 2 above)
Comment (lwsanty) [reply sql/migrations/080_jira_issue_type.up.sql]: Done.

**Review by dennwc — APPROVED** [review]

**Pull request merged**